commit 57736ae61cc3ffbdff5ff2af507da9f3aac8f924 (HEAD -> master)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-03-18T16:30:12+00:00 (7 months ago)
    
    TRITON-884 remove cloud analytics (ca) from Triton
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/README.md b/README.md
index db6a71a..cdc062b 100644
--- a/README.md
+++ b/README.md
@@ -10,6 +10,8 @@
 
 # sdc-cloud-analytics
 
+***NOTICE: the code in this repository is no longer in use. See TRITON-884.***
+
 This repository is part of the Joyent SmartDataCenter project (SDC).  For 
 contribution guidelines, issues, and general documentation, visit the main
 [SDC](http://github.com/joyent/sdc) project page.
